lib/benchpark/experiment.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, Tuple
+import yaml  # TODO: some way to ensure yaml available
+
+from benchpark.directives import SpecTemplate
+import benchpark.spec
+import benchpark.paths
+import benchpark.repo
+import benchpark.runtime
+import benchpark.variant
+
+import ramble.language.language_base
+import ramble.language.language_helpers
+
+bootstrapper = benchpark.runtime.RuntimeResources(benchpark.paths.benchpark_home)
+bootstrapper.bootstrap()
```

Comment:
```suggestion
bootstrapper = benchpark.runtime.RuntimeResources(benchpark.paths.benchpark_home)
bootstrapper.bootstrap()

import ramble.language.language_base
import ramble.language.language_helpers
```
Need to bootstrap before we import from ramble